main-module/src/main/java/com/diffblue/javatest/StringStuff.java

```diff
@@ -0,0 +1,346 @@
+package com.diffblue.javatest;
+
+import java.io.PrintStream;
+import java.util.Locale;
+
+public class StringStuff {
+
+  public static String classGetNameExample(Class c) {
```

Comment:
I'd say let's leave it as it is for now. If this becomes a problem let's modify it.